modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,276 @@
+.. On the *Accounts and roles* page, click *Refresh ARNs* and verify that the installer, support, worker, and control plane account roles are detected.
+
+. Select *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. This option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. This option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Optional: Select *Encrypt persistent volumes with customer keys* if you want to provide your own AWS Key Management Service (KMS) key Amazon Resource Name (ARN). The key is used for encryption of persistent volumes in your cluster.
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type*.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type in the default machine pool. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Configure autoscaling for the default machine pool:
+.. Select *Enable autoscaling* to automatically scale the number of machines in your default machine pool to meet the deployment needs.
+.. Set the minimum and maximum node count limits for autoscaling. The cluster autoscaler does not reduce or increase the default machine pool node count beyond the limits that you specify.
+** If you deployed your cluster using a single availability zone, set the *Minimum node count* and *Maximum node count*. This defines the minimum and maximum compute node limits in the availability zone.
+** If you deployed your cluster using multiple availability zones, set the *Minimum nodes per zone* and *Maximum nodes per zone*. This defines the minimum and maximum compute node limits per zone.
++
+[NOTE]
```

Comment:
Should this note be indented with the second bullet like it is now, or should it be indented a little more left with the text from 9b?

Reply:
My preference is for the note to align more to the left, with 9b rather than the bullet point nested in that. I don't know how to move the alignment to the left and still have the note under the bullet point. I will look into this.

Reply:
I added a wrapping complex around the substeps in step 9, which has enabled me to move the note indentation in line with 9b. I have not pushed the change to the PR yet, because I'm going through the remaining review suggestions first.

Reply:
Resolved, through the use of a wrapping complex.